lib/gds_api/test_helpers/router.rb

```diff
@@ -5,6 +5,35 @@ module TestHelpers
     module Router
       ROUTER_API_ENDPOINT = Plek.current.find('router-api')
 
+      def stub_router_has_route(path, route, bearer_token = nil)
+        bearer_token = ENV['ROUTER_API_BEARER_TOKEN'] unless bearer_token
```

Comment:
On the assumption that people might want to deliberately stub with a `nil` bearer token I'd suggest this:
```
def stub_router_has_route(path, route, bearer_token = ENV['ROUTER_API_BEARER_TOKEN'])
  stub_get_route # etc
```

Reply:
Sounds reasonable, I also realised I had put that configuration in the wrong place... so that's fixed now too!